extension/src/extension/types.ts

```diff
@@ -1,22 +1,10 @@
-export enum MsgType {
-    PageContent = 'page-content',
-    PopUpInit = 'popup-init',
-};
-
-export type ContentToBackgroundMsg = {
-    type: MsgType,
-    text: string
-};
-
-export type PopupToBackgroundMsg = {
-    type: MsgType
-};
-
-export type BackgroundToPopupMsg = {
-    articleTitle: string,
-    textToSummarize: string
-};
+export type MLISRequest = {
+    url: string,
+}
 
 export type MLISResponse = {
-    text: string
+    articleSummary: string,
```

Comment:
we can encompass different types of responses if we make `articleSummary` nullable and also add a nullable string field `url`.

Reply:
Good point!
I made everything nullable and changed `url` to `message`.
I don't think we need to retrieve `url` from failure responses, but we could at least use failure messages.
Please see changes to `lambda_function.py` attached to this PR and let me know if you have other opinions 😀